MealPlanner/Models/Database.cs

```diff
@@ -96,5 +99,41 @@ public static bool AddBodyStats(User u)
                 return false;
             }
         }
+        public static bool ValidateUser(string userName, string Password)
+        {
+            using (SqlConnection con = new SqlConnection(ConfigurationManager.ConnectionStrings["MealPlanDatabaseConnection"].ConnectionString))
+            {
+                con.Open();
```

Comment:
Do we need `con.Close()` or `con.Complete()`?

Reply:
Since it uses "using", which means it will close the connection automatically, if not use 'using', then it should close manually.